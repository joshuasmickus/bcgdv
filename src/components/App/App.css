.app {
  display: block;
  -moz-box-sizing:border-box;
  -webkit-box-sizing:border-box;
  box-sizing: border-box;
  width: 100%;
  margin: 0;
  padding: 20px;
}
