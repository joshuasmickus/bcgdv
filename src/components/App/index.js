import './App.css';

export { App as default } from './App';
