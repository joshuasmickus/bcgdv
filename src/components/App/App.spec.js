import React from 'react';
import { shallow } from 'enzyme';

import { App } from './App';

describe('Given an app component', () => {
  describe('when it is rendered', () => {
    it('should match the snapshot', () => {
      const wrapper = shallow(<App />);

      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
