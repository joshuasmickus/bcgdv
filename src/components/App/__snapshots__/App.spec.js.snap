// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given an app component when it is rendered should match the snapshot 1`] = `
<div
  className="app"
>
  <Connect(Board) />
</div>
`;
